src/XRP/serializer.ts

```diff
@@ -7,7 +7,7 @@ import {
 } from './generated/org/xrpl/rpc/v1/transaction_pb'
 
 interface PaymentJSON {
-  Amount: object | string
+  Amount: Record<string, unknown> | string
```

Comment:
Avoid using the object type. [See reasoning here](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-types.md#default-options).